### PR TITLE
fix bug with update group

### PR DESCRIPTION
**Pull request by NikitaSedyx**

### CODE REVIEW

File: items/resources.py [hunk 1]
```diff
@@ -116,6 +116,16 @@ class Meta:
     paginator_class = Paginator
     limit = 10
 
+  def hydrate_users(self, bundle):
+    body = json.loads(bundle.request.body)
+    bundle.obj.users.clear()
+    for user in body['users']:
```
Comment (ghost) [inline items/resources.py]: what if `users` is absent in `body`?

File: items/resources.py [hunk 2]
```diff
@@ -116,6 +116,16 @@ class Meta:
...
+      user = User.objects.get(pk=user['id'])
+      bundle.obj.users.add(user)
+    if (bundle.obj.creator not in bundle.obj.users.all()):
+      bundle.obj.users.add(bundle.obj.creator)
```
Comment (ghost) [inline items/resources.py]: this is strange, why do you need creator in `users`?

Comment (NikitaSedyx) [inline items/resources.py]: because creator is contributor too
and in GroupAuth it will be check users

File: items/resources.py (hunk 1 above)
Comment (NikitaSedyx) [inline items/resources.py]: it will be error with 500

File: items/resources.py (hunk 2 above)
Comment (ghost) [inline items/resources.py]: maybe it's better to change `GroupAuth` then? to enable doing things for creator personally, no?

Comment (NikitaSedyx) [inline items/resources.py]: I don't know, mb yes

**Comment by ghost** [comment]
:+1:

**Pull request merged**